lib/services/ngsi/ngsiService.js

```diff
@@ -428,7 +428,7 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
     var payload = {};
 
     var options = createRequestObject(
-        '/v2/entities/' + entityName + '/attrs',
+        '/v2/entities/' + entityName + '/attrs' + '?type=' + typeInformation.type,
```

Comment:
Maybe check if type is in typeInformation?

Reply:
Do you mean, for example, initialize the var ``options`` after the ``if`` in https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/ngsi/ngsiService.js#L439 ?